docs/docs/services.md

```diff
@@ -725,3 +725,300 @@ const createPost = (input) => {
 ```
 
 This makes sure that the user that's logged in and creating the post cannot reuse the same blog post title as one of their own posts.
+
+## Caching
```

Comment:
Do you think we could move this to another doc Rob? I think at some point we'll need to cleanup the services docs and explain a bit more what they do.

Reply:
Hmm, I would at some point, if we actually get some more cache-related things, but as it stands right now I couldn't find any dedicated docs for GraphQL caching, I think DT just made an in-depth discussion forum post. Since this cache mechanism is dedicated to Services I'd leave it there for now.